Deployment fails

I just tried a couple of deploys, but both times it failed:

![image](https://cloud.githubusercontent.com/assets/2734580/14901141/e11cb2ec-0d8b-11e6-9225-f5bc260d78b0.png)

Are the any logs so I can try to help find a fix?

These are the screenshots from the Azure Portal:

![image](https://cloud.githubusercontent.com/assets/2734580/14901183/1b2f64e8-0d8c-11e6-9dd7-3e9ed1379bf3.png)

![image](https://cloud.githubusercontent.com/assets/2734580/14901201/278e5960-0d8c-11e6-9a74-adae22abfa51.png)

Seems the errors all start with:

```
Selected npm version 3.5.1
npm ERR! fetch failed https://registry.npmjs.org/webidl-conversions/-/webidl-conversions-2.0.1.tgz
npm WARN retry will retry, error on last attempt: Error: connect EACCES 23.235.47.162:443 - Local (undefined:undefined)
```

I wonder if something changed on npm that messes it up. I'm guessing that trying to run npm install manually in site\wwwroot from Kudu console would fail in the same way?

Not sure what is going on, but I just did a Standard w/ Medium size worker and got this..  Sits on building for quite some time until it finally failed.

```
Omitting next output lines...
The iisnode.yml file explicitly sets nodeProcessCommandLine. Automatic node.js version selection is turned off.
Selected npm version 3.5.1
npm ERR! Windows_NT 6.2.9200
npm ERR! argv "D:\\Program Files (x86)\\nodejs\\4.2.3\\node.exe" "D:\\Program Files (x86)\\npm\\3.5.1\\node_modules\\npm\\bin\\npm-cli.js" "install" "--production"
npm ERR! node v4.2.3
npm ERR! npm  v3.5.1
npm ERR! code ECONNRESET
npm ERR! errno ECONNRESET
npm ERR! syscall read

npm ERR! network read ECONNRESET
npm ERR! network This is most likely not a problem with npm itself
npm ERR! network and is related to network connectivity.
npm ERR! network In most cases you are behind a proxy or have bad network settings.
npm ERR! network 
npm ERR! network If you are behind a proxy, please make sure that the
npm ERR! network 'proxy' config is set properly.  See: 'npm help config'
npm ERR! Windows_NT 6.2.9200
npm ERR! argv "D:\\Program Files (x86)\\nodejs\\4.2.3\\node.exe" "D:\\Program Files (x86)\\npm\\3.5.1\\node_modules\\npm\\bin\\npm-cli.js" "install" "--production"
npm ERR! node v4.2.3
npm ERR! npm  v3.5.1
npm ERR! code ECONNRESET
npm ERR! errno ECONNRESET
npm ERR! syscall read

npm ERR! network read ECONNRESET
npm ERR! network This is most likely not a problem with npm itself
npm ERR! network and is related to network connectivity.
npm ERR! network In most cases you are behind a proxy or have bad network settings.
npm ERR! network 
npm ERR! network If you are behind a proxy, please make sure that the
npm ERR! network 'proxy' config is set properly.  See: 'npm help config'
npm ERR! Windows_NT 6.2.9200
npm ERR! argv "D:\\Program Files (x86)\\nodejs\\4.2.3\\node.exe" "D:\\Program Files (x86)\\npm\\3.5.1\\node_modules\\npm\\bin\\npm-cli.js" "install" "--production"
npm ERR! node v4.2.3
npm ERR! npm  v3.5.1
npm ERR! code ECONNRESET
npm ERR! errno ECONNRESET
npm ERR! syscall read

npm ERR! network read ECONNRESET

...
ailed exitCode=1, command="D:\Program Files (x86)\nodejs\4.2.3\node.exe" "D:\Program Files (x86)\npm\3.5.1\node_modules\npm\bin\npm-cli.js" install --production
An error has occurred during web site deployment.
npm ERR! Windows_NT 6.2.9200\r\nnpm ERR! argv "D:\\Program Files (x86)\\nodejs\\4.2.3\\node.exe" "D:\\Program Files (x86)\\npm\\3.5.1\\node_modules\\npm\\bin\\npm-cli.js" "install" "--production"\r\nnpm ERR! node v4.2.3\r\nnpm ERR! npm  v3.5.1\r\nnpm ERR! code ECONNRESET\r\nnpm ERR! errno ECONNRESET\r\nnpm ERR! syscall read\r\n\r\nnpm ERR! network read ECONNRESET\r\nnpm ERR! network This is most likely not a problem with npm itself\r\nnpm ERR! network and is related to network connectivity.\r\nnpm ERR! network In most cases you are behind a proxy or have bad network settings.\r\nnpm ERR! network \r\nnpm ERR! network If you are behind a proxy, please make sure that the\r\nnpm ERR! network 'proxy' config is set properly.  See: 'npm help config'\r\nnpm ERR! Windows_NT 6.2.9200\r\nnpm ERR! argv "D:\\Program Files (x86)\\nodejs\\4.2.3\\node.exe" "D:\\Program Files (x86)\\npm\\3.5.1\\node_modules\\npm\\bin\\npm-cli.js" "install" "--production"\r\nnpm ERR! node v4.2.3\r\nnpm ERR! npm  v3.5.1\r\nnpm ERR! code ECONNRESET\r\nnpm ERR! errno ECONNRESET\r\nnpm ERR! syscall read\r\n\r\nnpm ERR! network read ECONNRESET\r\nnpm ERR! network This is most likely not a problem with npm itself\r\nnpm ERR! network and is related to network connectivity.\r\nnpm ERR! network In most cases you are behind a proxy or have bad network settings.\r\nnpm ERR! network \r\nnpm ERR! network If you are behind a proxy, please make sure that the\r\nnpm ERR! network 'proxy' config is set properly.  See: 'npm help config'\r\nnpm ERR! Windows_NT 6.2.9200\r\nnpm ERR! argv "D:\\Program Files (x86)\\nodejs\\4.2.3\\node.exe" "D:\\Program Files (x86)\\npm\\3.5.1\\node_modules\\npm\\bin\\npm-cli.js" "install" "--production"\r\nnpm ERR! node v4.2.3\r\nnpm ERR! npm  v3.5.1\r\nnpm ERR! code ECONNRESET\r\nnpm ERR! errno ECONNRESET\r\nnpm ERR! syscall read\r\n\r\nnpm ERR! network read ECONNRESET\r\nnpm ERR! network This is most likely not a problem with npm itself\r\nnpm ERR! network and is related to network connectivity.\r\nnpm ERR! network In most cases you are behind a proxy or have bad network settings.\r\nnpm ERR! network \r\nnpm ERR! network If you are behind a proxy, please make sure that the\r\nnpm ERR! network 'proxy' config is set properly.  See: 'npm help config'\r\nnpm ERR! Windows_NT 6.2.9200\r\nnpm ERR! argv "D:\\Program Files (x86)\\nodejs\\4.2.3\\node.exe" "D:\\Program Files (x86)\\npm\\3.5.1\\node_modules\\npm\\bin\\npm-cli.js" "install" "--production"\r\nnpm ERR! node v4.2.3\r\nnpm ERR! npm  v3.5.1\r\n\r\nnpm ERR! Callback called more than once.\r\nnpm ERR! \r\nnpm ERR! If you need help, you may report this error at:\r\nnpm ERR!     <https://github.com/npm/npm/issues>\r\n\r\nnpm ERR! Please include the following file with any support request:\r\nnpm ERR!     D:\home\site\wwwroot\npm-debug.log\r\nC:\Program Files (x86)\SiteExtensions\Kudu\54.50506.2209\bin\scripts\starter.cmd "D:\home\site\deployments\tools\deploy.cmd"
```

--- a/iisnode.yml
+++ b/iisnode.yml
@@ -2,4 +2,3 @@ node_env: production
 loggingEnabled: true
 logDirectory: iisnode
 enableXFF: true
-nodeProcessCommandLine: "D:\Program Files (x86)\nodejs\4.4.1\node.exe"

